lang/en/auth_saml2.php

```diff
@@ -140,3 +140,13 @@
  * Privacy provider (GDPR)
  */
 $string["privacy:no_data_reason"] = "The Saml2 authentication plugin does not store any personal data.";
+
+/*
+ * Signing Algorithm
+ */
+$string['sha1'] = 'Legacy SHA1 (Dangerous)';
```

Comment:
"(Dangerous)" love it :) :+1:

Reply:
I would love to take credit for that, but it was @catalystfd who put that into the Mahara version:

https://github.com/MaharaProject/mahara/commit/059b0765e4ce6867cccfda3c6b1d5426fe2c878d#diff-22db7a49684162f53954690a821df437R83